### ISSUE TITLE
Clarification for pretrained models

### ISSUE THREAD

**Issue by andreas128** [xinntao/BasicSR#13]
Are following statements correct?

- `sft_net_ini.pth` is a model that was trained for L1 loss only on DIV2K (+probability maps for each of the 800 DIV2K images)
- ImageNet/Flickr pictures were not used to training `sft_net_ini.pth` or any SFTGAN model.

**Comment by xinntao** [comment]
@andreas128 
1. We first trained a general SR model using the L1 loss on DIV2K and then we inserted initialized SFT layers to the mode and got the `sft_net_ini.pth`. So the `sft_net_ini.pth` did not use the probability maps.
2. In this pytorch version, we do not use ImageNet/Flickr pictures.

**Comment by xinntao** [comment]
@andreas128 
Sorry that I have made a mistake.
We first trained a SRGAN model on IDIV2K, i.e., using **VGG and GAN loss** and then we inserted initialized SFT layers to the mode and got the `sft_net_ini.pth`. And the `sft_net_ini.pth` did not use the probability maps.

**Comment by andreas128** [comment]
Thanks!
1) What is IDIV2K?
2) So you trained SRGAN from scratch without pretraining for PSNR only?
3) Could you please also upload the Discriminators too? It would come in handy as I'd like to fine tune the network to a different task. (For SFTGAN_bicx4_noBN_OST_bg.pth)

**Comment by xinntao** [comment]
@andreas128 
1. a typo mistake, is DIV2K
2. We train PSNR-SR --> SRGAN --> SFTGAN.
3. In my experiments, I only used the pre-trained generator and train the discriminator from scratch. 
I attached two discriminators (Maybe they are not optimal as I cannot find the original models) [GoogleDrive](https://drive.google.com/open?id=17_uX9c5sj64u2QDYHNmtaVr7LtlGj1r6).
